lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -3157,6 +3157,15 @@ bool TypeSystemSwiftTypeRef::IsTypedefType(opaque_compiler_type_t type) {
     return node && node->getKind() == Node::Kind::TypeAlias;
   };
 
+#ifndef NDEBUG
+  // We skip validation when dealing with a builtin type since builtins are
+  // considered type aliases by Swift, which we're deviating from.
```

Comment:
`... since SwiftASTContext reconstructs Builtin types as TypeAliases pointing to the actual Builtin types, but mangled names always describe the underlying builtins directly.`